<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransactionReceive</class>
 <widget class="QWidget" name="FileTransactionReceive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="primaryFrame" native="true">
       <property name="minimumSize">
        <size>
         <width>778</width>
         <height>430</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>778</width>
         <height>430</height>
        </size>
       </property>
       <widget class="control::MwcPushButtonNormal" name="processButton">
        <property name="geometry">
         <rect>
          <x>414</x>
          <y>390</y>
          <width>175</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Generate Response</string>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="lockHeightLabel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>179</y>
          <width>500</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>3874868</string>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="mwcLabel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>110</y>
          <width>500</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>2355 MWC</string>
        </property>
       </widget>
       <widget class="control::MwcLabelLarge" name="transactionType">
        <property name="geometry">
         <rect>
          <x>264</x>
          <y>33</y>
          <width>388</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Receive Transaction</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="label_5">
        <property name="geometry">
         <rect>
          <x>104</x>
          <y>143</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;b&gt;Transaction ID:&lt;/b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="senderDescLabel">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>243</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;b&gt;Sender's Description:&lt;/b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>3</y>
          <width>52</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources_desktop.qrc">:/img/FinalizeL@2x.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="label_6">
        <property name="geometry">
         <rect>
          <x>104</x>
          <y>179</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;b&gt;Lock Height:&lt;/b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="label_3">
        <property name="geometry">
         <rect>
          <x>104</x>
          <y>113</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;b&gt;Amount:&lt;/b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="transactionIdLabel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>146</y>
          <width>500</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>45983594</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="cancelButton">
        <property name="geometry">
         <rect>
          <x>195</x>
          <y>390</y>
          <width>175</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="receiverMessage">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>290</y>
          <width>500</width>
          <height>76</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="toolTip">
         <string>You can type message here for sender</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.SF NS Text';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Description (optional)</string>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="receiverLabel">
        <property name="geometry">
         <rect>
          <x>84</x>
          <y>211</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;b&gt;Sender Address:&lt;/b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="receiverAddressLabel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>211</y>
          <width>500</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>XXXXXXXXXX</string>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="senderMessage">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>246</y>
          <width>500</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>XXXXXXXXXX</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="receiverDescLabel">
        <property name="geometry">
         <rect>
          <x>84</x>
          <y>290</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;b&gt;My Description:&lt;/b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="progressFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="control::MwcLabelProgress" name="progress">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>PR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelProgress</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabelProgress.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>receiverMessage</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>processButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources_desktop.qrc"/>
 </resources>
 <connections/>
</ui>
